kani-compiler/src/codegen_cprover_gotoc/compiler_interface.rs

```diff
@@ -302,10 +302,10 @@ fn check_crate_items(gcx: &GotocCtx) {
                 );
                 tcx.sess.err(&error_msg);
             } else {
-                warn!(
+                tcx.sess.warn(format!(
                     "Ignoring global ASM in crate {}. Verification results may be impacted.",
```

Comment:
Please use `{gcx.short_crate_name()}` instead, which will be necessary to make clippy happy with the upcoming rust toolchain update.

Reply:
oh, I didn't know you couldn't use expressions. We'll do

Reply:
It's not working for me. I get the following error:
```
error: invalid format string: expected `'}'`, found `'.'`
   --> kani-compiler/src/codegen_cprover_gotoc/compiler_interface.rs:314:55
    |
314 |                     "Ignoring global ASM in crate {gcx.short_crate_name()}. Verification results may be impacted.",
    |                                                   -   ^ expected `}` in format string
    |                                                   |
    |                                                   because of this opening brace
    |
    = note: if you intended to print `{`, you can escape it using `{{`
```

Reply:
Sorry, I was wrong. Expressions cannot be used.